Gemfile

```diff
@@ -2,4 +2,6 @@ source "http://rubygems.org"
 
 gemspec
 
-File.exist?(gemfile_local = File.expand_path('../Gemfile.local', __FILE__)) and eval File.read(gemfile_local), binding, gemfile_local
+gemfile_local = File.expand_path('../Gemfile.local', __FILE__)
+
+File.exist?(gemfile_local) and eval File.read(gemfile_local), binding, gemfile_local
```

Comment:
Yeah, I suppose having one cryptic line isn't the best idea.  But in that case, how about cleaning it up even more?

```ruby
gemfile_local = File.expand_path '../Gemfile.local', __FILE__
eval File.read(gemfile_local), binding, gemfile_local if File.exist? gemfile_local
```